hbase-client/src/main/java/org/apache/hadoop/hbase/filter/FuzzyRowFilter.java

```diff
@@ -200,7 +200,7 @@ public boolean filterRow() throws IOException {
 
   @Override
   public ReturnCode filterCell(final Cell c) {
-    final int startIndex = lastFoundIndex >= 0 ? lastFoundIndex : 0;
+    final int startIndex = Math.max(lastFoundIndex, 0);
```

Comment:
Many thanks. :+1: 
Yes, this line change was not because of performance. Actually here just IntelliJ IDEA suggested to replace this to make it simpler to understand. It can be reverted if needed.